firstvoices/backend/views/storypage_views.py

```diff
@@ -102,15 +105,36 @@
     ),
 )
 class StoryPageViewSet(SiteContentViewSetMixin, FVPermissionViewSetMixin, ModelViewSet):
-    def get_detail_queryset(self):
+    def get_queryset(self):
         site = self.get_validated_site()
         story = self.get_validated_story(site)
-        return StoryPage.objects.filter(site=site, story=story).all()
-
-    def get_list_queryset(self):
-        site = self.get_validated_site()
-        story = self.get_validated_story(site)
-        return StoryPage.objects.filter(site=site, story=story).order_by("id").all()
+        user = self.request.user
+        return (
+            StoryPage.objects.filter(site=site, story=story)
```

Comment:
Opted not to use the `get_media_prefetch_list` util here as per ticket specifications we only want to order storypages related media by "created" for now.